### ISSUE TITLE
Refreshing page breaks the rendering

### ISSUE THREAD

**Issue by simoncarbajal** [fastify/fastify-swagger-ui#163]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.28.1

### Plugin version

4.0.0

### Node.js version

21.4.0

### Operating system

macOS

### Operating system version (i.e. 20.04, 11.3, 10)

Sonoma 14.5

### Description

It works, I can open and see each endpoint and I can execute them, they work!
![Screenshot 2024-07-19 at 10 53 56](https://github.com/user-attachments/assets/7cbb53c2-c2f1-44df-8e12-e4720b7ee2d4)
The page can always be refreshed (cmd + r, or place the cursor in the browser's url and press enter) being in the base url `http://localhost:3002/api/`.

But, if I open any endpoint (exactly like my previous screenshot) to see the documentation and then I refresh the page, I end up with this broken swagger:
![Screenshot 2024-07-19 at 10 54 36](https://github.com/user-attachments/assets/aff22a86-ecd3-4c95-a518-74d010185f3e)

```javascript
import swagger from '@fastify/swagger'
import ui from '@fastify/swagger-ui'
import { host, port } from '#api/env.js'
import { nodeEnv } from '#env.js'

export default async fastify => {
    const docsRoute = 'api'

    await fastify.register(swagger, {
        routePrefix: docsRoute,
        openapi: {
            openapi: '3.0.0',
            info: {
                title: 'My API',
                version: '0.0.1'
            },
            servers: [
                { url: 'http://localhost:3002', description: 'DEV' }
            ]
        },
        exposeRoute: true
    })

    await fastify.register(ui, {
        routePrefix: docsRoute,
        uiConfig: {
            docExpansion: 'none',
            deepLinking: true
        },
        exposeRoute: true
    })

    const isDev = nodeEnv === 'development'
    const schema = isDev ? 'http' : 'https'
    await fastify.ready()
    console.log('DOCUMENTATION:', `${schema}://${host}:${port}/${docsRoute}`)
}
```

### Expected Behavior

That I continue to see the same page instead of a broken one

**Comment by mcollina** [comment]
Thanks for reporting! Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by blue86321** [comment]
It's already fixed at #158

**Comment by SimonTweddle1** [comment]
I can confirm this issue is fixed but v4.0.1 is now broken for me with this [commit](https://github.com/fastify/fastify-swagger-ui/commit/8796c4c64d26f74c52a2e486cbf5fc99bf9e3a59). Doesn't load the page at all due to these errors:
![Screenshot 2024-07-26 at 09 27 02](https://github.com/user-attachments/assets/666b0104-062d-490a-8a75-2451ed17088d)
Removing the dot in the urls fixes the issue
